### DISCUSSION TITLE
제페토 스크립트 샘플 지원 건의 드립니다..

### DISCUSSION THREAD

**Discussion by mayalhc** [naverz/zepeto-studio-kor#140] [4. General]
안녕하세요..제페토 크리에이터 참이슬 입니다..
월드 출시 이후 많은 테스트를 해오면서 느끼는 것은 타입 스크립트는 유니티 엔진 개발하던 분들도 생소한 스크립트입니다..
물론 잘하는 분들도 계시겠지만 제페토 월드 제작 기본 상호 작용 스크립트 샘플 정도는 오픈 해주셔도 되지 않을까요?
현제 제페토 공식 월드에서 이루어지고 있는 샘플신 몇개 오픈 해주시면 많은 월드 제작 하는 분들이 참고하여 더 나은 
월드 제작에 많은 도움이 될것입니다...

현제 오픈된 샘플에는 월드 제작에 필요한 기본 스크립트 상호 작용 부분은 모두 제외 하고 공개하는 것은 아무 도움이 되지 않습니다.
아니면 기존 C# 프로그램을 사용할수 있게 해주시면 간단하게 해결될 문제 일수도있습니다..

전체 공개가 힘드시면 아주간단한 기능 몇가지만이라도 샘플로 공개 해주시면 앞으로 월드 제작에 많은 도움이 될것입니다.

**Comment by x4cf** [comment]
부족한 자료지만 제가 만든 월드의 모든 코드를 공개하고, 1월 중으로 튜토리얼 가이드를 배포할 예정입니다.
이전까지는 깃허브에서 질의응답을 받거나 저희와 소통하면서 문제를 해결해나가시는건 어떨까요?

코드 : Me793a2
https://open.kakao.com/o/gNaIjUKd

**Reply by runtostar** [reply]
안녕하세요. 
감사합니다. 좋은 아이디어인 것 같아요. 
근데 https://open.kakao.com/o/gNaIjUKd 가 없는 공개방으로 나오는데, 주소 확인 좀 부탁드립니다.

**Reply by x4cf** [reply]
https://open.kakao.com/o/sHLhciId
이곳으로 연락주시겠어요?

**Comment by hyeonjin-jo** [comment]
안녕하세요.

좋은 의견 감사드립니다.

크리에이터분들에게 제공할 샘플예제의 경우 내부적으로 논의되고 있습니다.

정확한 시점은 공유드리기 어려운 점 양해 부탁드립니다.

C#언어 사용이 어려운 점은 아래 링크의 답변에서 확인 가능합니다.

https://github.com/naverz/zepeto-studio-kor/discussions/138

감사합니다.
